packages/main/test/samples/Dialog.sample.html

```diff
@@ -158,4 +158,43 @@ <h3>Draggable and Resizable Dialog</h3>
 <ui5-dialog draggable resizable></ui5-dialog>
 	</xmp></pre>
 </section>
+
+<section>
+	<h3>Dialog with SAP Fiori Styled Footer</h3>
+	<div class="snippet">
+		<ui5-button id="fiori-footer">Open dialog</ui5-button>
+
+		<ui5-dialog id="fiori-footer-dialog" header-text="SAP Fiori Footer Buttons">
+			<p>Adding custom styles to achieve the look and feel of a SAP Fiori footer</p>
+			<div slot="footer" style="display: flex; align-items: center; justify-content: end; width: 100%; box-sizing: border-box;">
+				<ui5-button design="Emphasized">OK</ui5-button>
+				<ui5-button id="fiori-close" style="margin: 0 0 0 0.5rem;">Close</ui5-button>
```

Comment:
Add the min width 4rem to these elements too.
The code wrapped in a `pre` tag is meant for reading - It doesn't get executed, but both code sections should be exactly the same.